understanding/21/pointer-cancellation.html

```diff
@@ -14,6 +14,8 @@ <h3>Up-Event activation or completion</h3>
 			<p>The most accessible way to incorporate pointer cancellation is to make activation occur on the up-event.</p>
       
         <p>Up-event activation refers to the activation of a target when the pointer is released. In a touchscreen interaction, when the finger touches a target, the up-event activation only occurs when the finger is lifted while still being within the target boundary. Similarly in mouse interaction, the up-event occurs when the mouse button is released while the cursor is still within the boundary of the initial target set when the mouse button was pressed.</p>
```

Comment:
Clarification:

- "the up-event activation only occurs when the finger is lifted while still being within the target boundary" is a bit misleading. Yes, the `click` event isn't fired if the finger went down onto the touchscreen, then moved outside of the boundary, and then went up. However, regardless of whether the finger is inside or outside of the boundary, the `touchend` event is fired at the original element. If an author was listening to `touchend`, then they'd be firing functionality regardless of where the finger was at the end when it was lifted - inside or outside the element won't matter then.